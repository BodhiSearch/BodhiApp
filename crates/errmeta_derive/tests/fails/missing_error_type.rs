#[derive(Debug, errmeta_derive::ErrorMeta)]
pub enum TestError {
  #[error_meta(code = "bad_request", status = 200)]
  BadRequest,
}

fn main() {}
