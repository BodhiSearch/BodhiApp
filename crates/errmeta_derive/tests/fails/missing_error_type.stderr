error: error_type not specified for variant 'BadRequest'
 --> tests/fails/missing_error_type.rs:1:17
  |
1 | #[derive(Debug, errmeta_derive::ErrorMeta)]
  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `errmeta_derive::ErrorMeta` (in Nightly builds, run with -Z macro-backtrace for more info)
